lib/cisco_node_utils/vxlan_vtep_vni.rb

```diff
@@ -104,12 +104,16 @@ def destroy
     #                      PROPERTIES                      #
     ########################################################
 
+    def ingress_replication_supported?
+      node.cmd_ref.supports?('vxlan_vtep_vni', 'ingress_replication')
+    end
+
     def ingress_replication
       config_get('vxlan_vtep_vni', 'ingress_replication', @get_args)
     end
 
     def remove_add_ingress_replication(protocol)
-      if ingress_replication.empty?
+      if ingress_replication.to_s.empty?
```

Comment:
Since `ingress_replication` has `kind: string` in the YAML entry, there should be no need for converting the result to a string